android/app/src/main/java/poke/rogue/helper/presentation/biome/detail/gym/BiomeGymFragment.kt

```diff
@@ -36,6 +36,9 @@ class BiomeGymFragment :
     private fun initObservers() {
         repeatOnStarted {
             viewModel.uiState.collect { state ->
+                state.gymPokemons.isEmpty().let {
+                    binding.biomeDetailEmpty.visibility = if (it) View.VISIBLE else View.GONE
```

Comment:
```suggestion
binding.biomeDetailEmpty.setVisible(it)
```

commonBindingAdapter 에 있는 setVisible 의 사용을 고려해볼 수 있겠네요!